Fix handling storage errors (#1213)
microsoft/appcenter-sdk-dotnet: CHANGELOG.md

## Version 2.6.2 （Under development）

### App Center

#### UWP/WPF/WinForms

> * **[Fix]** Fix an issue where internal SDK exceptions would sometimes be "unobserved" and create noise as a result.

Reviewer: ```suggestion
* **[Fix]** Fix an issue where internal SDK exceptions would sometimes be "unobserved" and would cause undefined behavior.
```
